docs/pr-coding-guidelines.md

```diff
@@ -144,10 +144,18 @@ We do not use frameworks. However we use libraries, such suitcss-components-grid
 
 ### Javascript
 
-The js folder contains two subfolders: `lib` contains all project source files and `vendor` contains all third party libraries. The files out of the two sub folders contain the compiled source code.
+The js folder contains two sub folders: `lib` contains all project source files and `vendor` contains all third party libraries. The files out of the two sub folders contain the compiled source code.
 
 > Note: All code changes should be made to the files in `lib`
 
+#### Linting
+
+For the consistency of our codebase, we use eslint with these rules: https://github.com/webcompat/webcompat.com/blob/master/.eslintrc. We also use Prettier for formatting our code. We have set up two recipes to check and fixe the codebase.
```

Comment:
It's a great start @magsout! :)
I'd suggest to add a few more links to the tools and docs as following:

```
To make it easier for everyone to contribute, we use [eslint](http://eslint.org/) to keep the code base steady. You can view the rules in our [ESLint config](https://github.com/webcompat/webcompat.com/blob/master/.eslintrc#L34). For further explanation of the rules, please have a look at the [ESLint documentation](http://eslint.org/docs/rules/). 

For the linting process, we use [Prettier](http://jlongster.com/A-Prettier-Formatter). 
```

I'm not 100% sure what `recipes` means in this context. Could you explain it to me please? :)